### PR TITLE
i18n(es): Update `from-nextjs`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `from-nextjs`

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx [hunk 1]
```diff
@@ -446,120 +446,120 @@ Aquí te mostramos cómo recrear eso en `src/pages/index.astro`, remplazando `ge
...
-}
+    export default function Home({ pokemons }) {
+        return (
+            <>
+                <ul className={`plain-list ${styles.pokeList}`}>
+                    {pokemons.map((pokemon) => (
+                        <li className={styles.pokemonListItem} key={pokemon.name}>
+                            <Link className={styles.pokemonContainer} as={`/pokemon/${pokemon.name}`} href="/pokemon/[name]">
+                                <p className={styles.pokemonId}>No. {pokemon.id}</p>
+                                <img className={styles.pokemonImage} src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`${pokemon.name} picture`}></img>
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx:461]:
```suggestion
                                <img className={styles.pokemonImage} src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`Foto de ${pokemon.name}`}></img>
```

File: src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx [hunk 2]
```diff
@@ -446,120 +446,120 @@ Aquí te mostramos cómo recrear eso en `src/pages/index.astro`, remplazando `ge
...
-```
+    ```astro title="src/pages/index.astro" "class" "</a>" "<a"
+    ---
+    ---
+    <ul class="plain-list pokeList">
+        {pokemons.map((pokemon) => (
+            <li class="pokemonListItem" key={pokemon.name}>
+                <a class="pokemonContainer" href={`/pokemon/${pokemon.name}`}>
+                    <p class="pokemonId">No. {pokemon.id}</p>
+                    <img class="pokemonImage" src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`${pokemon.name} picture`}/>
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx:515]:
```suggestion
                    <img class="pokemonImage" src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`Foto de ${pokemon.name}`}/>
```

File: src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx [hunk 3]
```diff
@@ -446,120 +446,120 @@ Aquí te mostramos cómo recrear eso en `src/pages/index.astro`, remplazando `ge
...
+    });
+    ---
+
+    <Layout>
+      <ul class="plain-list pokeList">
+          {pokemons.map((pokemon) => (
+              <li class="pokemonListItem" key={pokemon.name}>
+                  <a class="pokemonContainer" href={`/pokemon/${pokemon.name}`}>
+                      <p class="pokemonId">No. {pokemon.id}</p>
+                      <img class="pokemonImage" src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`${pokemon.name} picture`}/>
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx:555]:
```suggestion
                      <img class="pokemonImage" src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon.id}.png`} alt={`Foto de ${pokemon.name}`}/>
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Gracias por el review @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good work 💜

**Pull request merged**